docs/firestore_document.js

```diff
@@ -48,6 +50,18 @@ function getDisasters() {
   return disasterCollectionReference().get();
 }
 
+/**
+ * @return {firebase.firestore.CollectionReference} all usershapes collection
+ */
+function userFeatures() {
+  return getFirestoreRoot().collection('usershapes');
+}
+
+/** @return {Promise<firebase.firestore.QuerySnapshot>} all user shapes */
+function getUserFeatures() {
+  return userFeatures().get();
```

Comment:
might as well follow the pattern of what I did in #414 and do what happens in `getDisastersData` below (can extract that code to a common function), so user gets back a map, rather than having to iterate themselves.